src/AutoRest.CSharp.V3/Generation/Writers/ManagementClientWriter.cs

```diff
@@ -113,7 +113,11 @@ public static void WriteAggregateClient(CodeWriter writer, BuildContext context)
 
                     using (writer.Scope())
                     {
-                        writer.WriteParameterNullChecks(allParameters.Values);
+                        foreach (var parameter in allParameters.Values)
```

Comment:
Why inline the method?

Reply:
Fixed as a part of https://github.com/Azure/autorest.csharp/pull/846#discussion_r461040400.